CHANGELOG.md

```diff
@@ -7,6 +7,7 @@
  - Actions secrets support with env vars (#2333)
  - Reduce the number of database connections used on startup
  - Validate required configuration keys blockstore.type, auth.encrypt.secret_key.  This *breaks* existing configurations that assume a default blockstore.type of "local".  But no such configuration may be production.
+ - lakeFS configuration load non-hidden config files first (#2355)
```

Comment:
Yes, but I also want *more*!  Can you give the list of locations where lakeFS searches for configuration, or alternatively link to the docs that gives the location?
```suggestion
 - Add search path to load lakeFS, see https://lakefs.io/docs/blah/blue (#2355)
```